docs/ScalarDLSupportedDatabase.md

```diff
@@ -0,0 +1,48 @@
+# Set up a Scalar DL supported database
```

Comment:
Is this a setup guide in general? or a specific guide with managed DBs for the other manual guide?
I assume the latter. If so, please describe so clearly. It's pretty unclear.

Reply:
Updated the sentence to `In this section, you will set up a database for Scalar DL.`. 
Should the heading also need to be changed?

Reply:
If it is a general guide, then it shouldn't talk about the custom config in helm since it makes the doc k8s specific.